### PR TITLE
@bentley/imodeljs-native 4.4.2

### PR DESCRIPTION
**Pull request by anmolshres98**

### CODE REVIEW

File: docs/changehistory/NextVersion.md [hunk 1]
```diff
@@ -2,4 +2,3 @@
 publish: false
 ---
 # NextVersion
-
```
Comment (aruniverse) [inline docs/changehistory/NextVersion.md:5]: just want to confirm there were no changes in nextver.md that got dropped when you created the merge

File: presentation/backend/src/presentation-backend/PresentationManager.ts [hunk 2]
```diff
@@ -319,6 +319,7 @@ export interface PresentationManagerProps {
    * data changes are not tracked at all.
    *
    * @beta
+   * @deprecated in 4.4. The manager now always tracks for iModel data changes without polling.
```
Comment (aruniverse) [inline presentation/backend/src/presentation-backend/PresentationManager.ts:322]: @mergifyio backport release/4.4.x

Comment (aruniverse) [reply presentation/backend/src/presentation-backend/PresentationManager.ts:322]: fyi @grigasp @saskliutas

**Comment by aruniverse** [comment]
@Mergifyio backport release/4.4.x

File: docs/changehistory/NextVersion.md (hunk 1 above)
Comment (DanRod1999) [reply docs/changehistory/NextVersion.md:5]: There was a change in next version.md from one of Affan's PRs. However, that should be included in nextVersion.md in the release/4.4.x branch. So as long as its there, 4.4.0.md will be properly documented

Comment (aruniverse) [reply docs/changehistory/NextVersion.md:5]: please make sure its there in the nextver.md in the 4.4. branch

Comment (anmolshres98) [reply docs/changehistory/NextVersion.md:5]: Yes, it has been incorporated in the other PR merging into release/4.4.x: https://github.com/iTwin/itwinjs-core/pull/6405/files#diff-4a7785af3b56bc69d2ad9f34257cd4a96cba53855aa039abeb761613d0f81988

**Review by aruniverse — APPROVED** [review]

**Review by grigasp — APPROVED** [review]

**Pull request merged**